INN/umbrella-mstoday
# Remove background color from scaip donate and signup interstitials as well

## Changes

This pull request makes the following changes:

- Removes the changes made in #78 in favor of just removing the background color everywhere instead of just on single posts

Homepage before:
![screencapture-mstoday-test-2020-05-26-09_36_35](https://user-images.githubusercontent.com/18353636/82907344-806d8b80-9f34-11ea-9584-6d6e39daaabc.png)

Homepage after:
![screencapture-mstoday-test-2020-05-26-09_35_26](https://user-images.githubusercontent.com/18353636/82907343-7fd4f500-9f34-11ea-8d83-e7f84cd7f560.png)

Single post before:
![screencapture-mstoday-test-2020-02-23-state-could-be-paying-10-percent-of-cost-instead-of-100-percent-for-some-community-mental-health-patients-2020-05-26-09_36_09](https://user-images.githubusercontent.com/18353636/82907342-7f3c5e80-9f34-11ea-9560-289c0dbba752.png)

Single post after:
![screencapture-mstoday-test-2020-02-23-state-could-be-paying-10-percent-of-cost-instead-of-100-percent-for-some-community-mental-health-patients-2020-05-26-09_35_50](https://user-images.githubusercontent.com/18353636/82907335-7ba8d780-9f34-11ea-9b88-6a700e4f65d0.png)

## Why



For #76

## Testing/Questions

Features that this PR affects:

- Donate interstitial, signup interstitial, and SCAIP widget areas on posts


Questions that need to be answered before merging:


Steps to test this PR:

1. View the homepage and a single post and make sure SCAIP widget areas have no background color
